CoEpi/api/Api.swift

```diff
@@ -10,7 +10,7 @@ protocol Api {
 
     func getCenKeys() -> Single<[String]>
 
-    func getCenReport(cenKey: CENKey) -> Single<CENReport>
+    func getCenReport(cenKey: CENKey, id: String, report: String) -> Single<CENReport>
```

Comment:
You just need the key to get the key to get the report, see https://github.com/Co-Epi/coepi-backend-go